domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/event/AbstractShowConditionValidator.java

```diff
@@ -62,6 +77,14 @@ protected void validateShowConditionFields(EventEntity eventEntity,
         }
     }
 
+    private boolean isInjectedField(String showConditionField, boolean isSupportingInjectedData) {
+        if (isSupportingInjectedData) {
+            return InjectedField.isInjectedField(showConditionField);
+        }
+
+        return false;
```

Comment:
```suggestion
        return isSupportingInjectedData && InjectedField.isInjectedField(showConditionField);
```

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/1203/commits/1a9c18091b83dd43eaf785f1bb6543fe7b24d22c